src/Tribe/Editor/Blocks/Rsvp.php

```diff
@@ -241,10 +241,11 @@ public function rsvp_process() {
 		if ( 0 === $ticket_id ) {
 			wp_send_json_error( $response );
 		}
-
-		$has_tickets = false;
-		$post_id     = get_the_id();
-		$ticket      = tribe( 'tickets.rsvp' )->get_ticket( $post_id, $ticket_id );
+		
+		$has_tickets    = false;
+		$event          = tribe( 'tickets.rsvp' )->get_event_for_ticket( $ticket_id );
+		$post_id        = $event->ID;
+		$ticket         = tribe( 'tickets.rsvp' )->get_ticket( $post_id, $ticket_id );
 
 		/**
 		 * RSVP specific action fired just before a RSVP-driven attendee tickets for an order are generated
```

Comment:
Can you also fix the `tribe( 'tickets.rsvp' )` usage to use `$rsvp` at line 257/267/272/319/321?